src/Steps/Step.php

```diff
@@ -114,7 +120,13 @@ public function runCommand(OutputInterface $output, $command, $error = null, $ex
 
         // Run it
         $process->setTimeout(null);
+
+        $verbosity = $output->getVerbosity();
+        if ($allowDebugVerbosity && $verbosity === OutputInterface::VERBOSITY_DEBUG) {
```

Comment:
```suggestion
        if ($allowDebugVerbosity && $output->isDebug()) {
```
The symfony check is a little different than just `===` - lets just ask symfony if it's debug instead of doing our own comparison.

Reply:
Updated